.github/workflows/template-validation.yml

```
@@ -14,53 +14,63 @@ env:
 
 jobs:
+  compute:
+    runs-on: ubuntu-latest
+    outputs:
+      py-min: ${{ steps.out.outputs.min }}
+      py-max: ${{ steps.out.outputs.max }}
+      py-matrix: ${{ steps.out.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v4
+      - id: out
+        uses: ./.github/actions/compute-python
```

actionlint (1.7.7)

description is required in metadata of "Compute Python min/max from pyproject" action at "/home/jailuser/git/.github/actions/compute-python/action.yml"

(action)